app/testContext/testContext.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import nunjucks from 'nunjucks';
+import { App } from '../../app';
+
+export const TetsContext = (macroWrapper, componentContext) => {

```

Comment:
Just a little unsure about the name. I would make it little more clear about the purpose of this function. So something like `createTestHarness`.

What do you think?

Reply:
or createDummyApp still as it hasn't changed in function